Update invite-your-team.adoc (#8486)
circleci/circleci-docs: jekyll/_cci2/invite-your-team.adoc

. A modal window will pop up. Click the btn:[Invite my team] button in the window to copy an invite message that you can send to your team members. The sign-up link is _not_ unique to a single team member. This link can be shared and used for multiple users at the same time.
. Your team members will need to use the sign-up link in your invite message to join your organization.

> To craft a custom invite link for your organization, go through the steps below. 

Reviewer: ```suggestion
Alternatively, to craft a custom invite link for your organization, follow these steps: 
```
